test/test-helpers/_setup.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+const fixtures = path.join(process.cwd(), 'test', 'fixtures');
+
+fs.renameSync(path.join(fixtures, 'repo-with-tags', 'git'), path.join(fixtures, 'repo-with-tags', '.git'));
```

Comment:
Maybe you can share the [foldernames](https://github.com/JPeer264/node-git-commit-info/blob/master/test.js#L10) in a special file. So you just have to add the foldername at one place, but affect the `_setup` and `_cleanup` file.